### DISCUSSION TITLE
Page not Found

### DISCUSSION THREAD

**Discussion by dreechu** [community/community#135728] [GitHub Education] [answered]
### Hi everyone,

I can't seem to access the github education benefits like digital ocean and such. It shows a 404 page when I open the site.
![image](https://github.com/user-attachments/assets/10241635-e241-4e59-8e7a-6812708c6195)
![image](https://github.com/user-attachments/assets/ecf38c2d-c660-4c04-b69d-3a42ddb1ff16)

**Comment by davevad93** [comment, marked as answer]
Hi @dreechu ,

Yes, today the `Student Developer Pack` offers page is inaccessible, I guess they will fix it soon.

**Reply by davevad93** [reply]
However, the link for the `Student Developer Pack` offers is working, check it out [here](https://education.github.com/pack), you can explore the available offers and access your benefits.
